apps/bfd-db-migrator/src/test/java/gov/cms/bfd/migrator/util/DatabaseTestUtils.java

```diff
@@ -0,0 +1,427 @@
+package gov.cms.bfd.migrator.util;
+
+import com.google.common.collect.ImmutableList;
+import gov.cms.bfd.sharedutils.exceptions.BadCodeMonkeyException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.UncheckedIOException;
+import java.io.Writer;
+import java.net.Inet4Address;
+import java.net.ServerSocket;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.flywaydb.core.Flyway;
+import org.hsqldb.jdbc.JDBCDataSource;
+import org.hsqldb.persist.HsqlProperties;
+import org.hsqldb.server.ServerAcl.AclFormatException;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Provides utilities for managing the database in integration tests.
+ *
+ * <p>Note: This is placed in <code>src/main/java</code> (rather than <code>src/test/java</code>)
+ * for convenience: test dependencies aren't transitive, which tends to eff things up.
```

Comment:
we should fix this (I know it is pre-existing) by introducing a maven shared testing utilities project. can we at least get a TODO and JIRA on it?

Reply:
yup I'll include this as part of the 1558 work and move it into a (small) common testing utils project